src/test/java/com/puppycrawl/tools/checkstyle/grammar/javadoc/GeneratedJavadocTokenTypesTest.java

```diff
@@ -46,7 +46,6 @@
     public void testTokenNumbers() {
         assertEquals(1, JavadocParser.LEADING_ASTERISK, MSG);
         assertEquals(2, JavadocParser.HTML_COMMENT_START, MSG);
-        assertEquals(3, JavadocParser.CDATA, MSG);
```

Comment:
@romani Should this remain in?

Reply:
@pbludov If field is still in JavadocParser, then we need to keep this check in.

Reply:
done.